generators/client/templates/react/src/main/webapp/app/config/constants.ts

```diff
@@ -9,3 +9,8 @@ export const messages = {
   DATA_DELETE_SUCCESS_ALERT: 'Data deleted successfully!',
   DATA_UPDATE_ERROR_ALERT: 'Data deleted successfully!'
 };
+
+export const APP_DATE_FORMAT = 'DD/MM/YY HH:mm';
+
+export const APP_FORMAT_NUMBER_WHOLE = '0,0';
```

Comment:
could you please change this to `APP_NUMBER_WHOLE_FORMAT` & `APP_NUMBER_TWO_DIGITS_AFTER_POINT_FORMAT` for consistency

Reply:
never mind ill change it

Reply:
ok I was ready to change ;)